spec/controllers/answer_tags_controller_spec.rb

```diff
@@ -3,6 +3,12 @@
   #factory objects required for "action_allowed" test cases
   let(:instructor) { build(:instructor, id: 1) }
   let(:student) { build(:student, id: 1) }
+  let!(:assignment) { create(:assignment, id: 1) }
+  let!(:assignment2) { create(:assignment, id: 2) }
+  let!(:questionnaire) { create(:questionnaire, id: 1) }
+  let!(:tag_prompt_deployment) { create(:tag_prompt_deployment, id: 1, assignment_id: 1, questionnaire_id: 1) }
+  let!(:tag_prompt_deployment2) { create(:tag_prompt_deployment, id: 2, assignment_id: 2, questionnaire_id: 1) }
```

Comment:
are assignment2 & tag_prompt_deployment2 being used?

Reply:
They are not explicitly called, but using `let!` adds the entries in the table which are eventually queried in the `index` method.